crates/astria-bridge-withdrawer/tests/blackbox/helpers/test_bridge_withdrawer.rs

```diff
@@ -237,7 +237,7 @@ impl TestBridgeWithdrawer {
     }
 }
 
-#[allow(clippy::module_name_repetitions)]
+#[expect(clippy::module_name_repetitions, reason = "naming is for clarity here")]
```

Comment:
I think we should actually fix this by renaming to `Config` so that this type can be identified by `test_bridge_withdrawer::Config`